src/main/java/de/triology/cas/services/CesServicesManagerStageProductive.java

```diff
@@ -1,6 +1,7 @@
 package de.triology.cas.services;
 
 import org.jasig.cas.services.RegisteredService;
+import org.json.simple.parser.ParseException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.json.simple.parser.ParseException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)